### ISSUE TITLE
Extension on marketplace is out of sync with this repo

### ISSUE THREAD

**Issue by a5ehren** [xaverh/vscode-clang-format#159]
The marketplace extension has not been updated since 2019, while this repo has been updated many times since then. Author needs to cut a new release or delete the extension.

**Comment by hansfn** [comment]
@xaverh Any hope of getting an update on the marketplace? It's a pity to fork it just to get a new release ...

**Comment by xinthose** [comment]
bump; you have 2 million downloads; you should probably fix it; thank you

**Comment by wbkboyer** [comment]
Thank you @xaverh for this excellent VSCode plugin! Like the above folks have requested, would it be possible to cut a new release (last was [1.6.1](https://github.com/xaverh/vscode-clang-format/releases/tag/1.6.1) cut in 2017) and to update the VSCode plugin (last update was in 2019)? Like @a5ehren mentioned, you've made a lot of updates, so it would be awesome to see those made available in this VSCode plugin! As well, the [ClangFormat](https://clang.llvm.org/docs/ClangFormat.html) documentation still lists your extension as the [recommended extension](https://clang.llvm.org/docs/ClangFormat.html#visual-studio-code-integration) for `clang-format`.

Thanks again!

**Comment by a5ehren** [comment]
> Thank you [@xaverh](https://github.com/xaverh) for this excellent VSCode plugin! Like the above folks have requested, would it be possible to cut a new release (last was [1.6.1](https://github.com/xaverh/vscode-clang-format/releases/tag/1.6.1) cut in 2017) and to update the VSCode plugin (last update was in 2019)? Like [@a5ehren](https://github.com/a5ehren) mentioned, you've made a lot of updates, so it would be awesome to see those made available in this VSCode plugin! As well, the [ClangFormat](https://clang.llvm.org/docs/ClangFormat.html) documentation still lists your extension as the [recommended extension](https://clang.llvm.org/docs/ClangFormat.html#visual-studio-code-integration) for `clang-format`.
> 
> Thanks again!

FWIW I ended up forking this repo and releasing my own extension as clang-format-2025. I need to bump the depends again but I think it is functionally complete.

**Comment by wbkboyer** [comment]
> > Thank you [@xaverh](https://github.com/xaverh) for this excellent VSCode plugin! Like the above folks have requested, would it be possible to cut a new release (last was [1.6.1](https://github.com/xaverh/vscode-clang-format/releases/tag/1.6.1) cut in 2017) and to update the VSCode plugin (last update was in 2019)? Like [@a5ehren](https://github.com/a5ehren) mentioned, you've made a lot of updates, so it would be awesome to see those made available in this VSCode plugin! As well, the [ClangFormat](https://clang.llvm.org/docs/ClangFormat.html) documentation still lists your extension as the [recommended extension](https://clang.llvm.org/docs/ClangFormat.html#visual-studio-code-integration) for `clang-format`.
> > Thanks again!
> 
> FWIW I ended up forking this repo and releasing my own extension as clang-format-2025. I need to bump the depends again but I think it is functionally complete.

Right on, thank you for your work to on your fork, @a5ehren !! I took a quick look, and I see there’s VSCode docs for how to install an extension from the .vsix file (see [here](https://code.visualstudio.com/docs/configure/extensions/extension-marketplace#_install-from-a-vsix)), then checked your repo and see that you are indeed building the .vsix and including it in the release (e.g. [clang-format-2025-2.0.9.vsix](https://github.com/a5ehren/vscode-clang-format/releases/download/v2.0.9/clang-format-2025-2.0.9.vsix) )! Hot dang!!

I still think there’s an important discussion with @xaverh to check to make sure this project is still being maintained, since this extension is the one recommended in the [ClangFormat](https://clang.llvm.org/docs/ClangFormat.html) documentation ❤️

**Comment by a5ehren** [comment]
Agree, I might send a doc update to the LLVM people to point to my extension instead.

**Comment by a5ehren** [comment]
Guess it's a good thing I looked. Msft had unpublished my fork for some reason, so I had to rename it. It's now "ebextensions.active-clang-format"
